packages/@featherds/checkbox/demos/Checkbox.vue

```diff
@@ -33,5 +34,5 @@ export default {
   components: {
```

Comment:
I know it wasn't added here, but can we convert the alert to be a console.log instead if it's to demonstrate something, as it's much less intrusive?

Reply:
fixed